keadm/cmd/keadm/app/cmd/debug/get_flags.go

```diff
@@ -104,6 +104,26 @@ func (f *HumanPrintFlags) AllowedFormats() []string {
 	return []string{"wide"}
 }
 
+// NewHumanPrintFlags returns flags associated with
+// human-readable printing, with default values set.
+func NewHumanPrintFlags() *HumanPrintFlags {
+	showLabels := false
+	sortBy := ""
+	showKind := false
+	columnLabels := []string{}
+
+	return &HumanPrintFlags{
+		NoHeaders:     false,
+		WithNamespace: false,
+		ColumnLabels:  &columnLabels,
+
+		Kind:       schema.GroupKind{},
+		ShowLabels: &showLabels,
+		SortBy:     &sortBy,
+		ShowKind:   &showKind,
+	}
+}
+
 // ToPrinter receives an outputFormat and returns a printer capable of
 // handling human-readable output.
 func (f *HumanPrintFlags) ToPrinter(outputFormat string) (printers.ResourcePrinter, error) {
```

Comment:
Please fix the lint error:

![image](https://github.com/kubeedge/kubeedge/assets/48788150/fee5ec0b-5f00-4d4a-b3fc-eb8426759598)